intelmq/bots/parsers/nothink/parser_dns_attacks.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+""" Parser for IntelMQ and Nothink honeypot dns attack feed """
+
+import sys
+
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.harmonization import FQDN
+from intelmq.lib.message import Event
+
+
+class NothinkDNSAttackParserBot(ParserBot):
+    """ Nothink DNS Attack Bot """
+
+    def parse_line(self, line, report):
+        if line.startswith('#') or len(line) == 0:
+            self.tempdata.append(line)
+
+        else:
+            line = line.strip('"').split('","')
+            event = Event(report)
+            event.add('time.source', line[0] + '+00:00')
+            event.add('source.ip', line[1])
+            event.add('source.asn', line[2])
+            event.add('source.as_name', line[3])
+            if FQDN.is_valid(line[4]):
+                event.add('source.reverse_dns', line[4])
+            if len(line[5]) == 2:
```

Comment:
Yes it looks that way as the other valid ones are two character.  Also for alpha-3 UNK is a code that is agreed not to be used.